Hybotix

hybotix opened the pull request:
Added the rest of the channels to the README.md file. No channel descriptions yet.

Commit:
Added the rest of the channels under the channel sections.

Commit:
Added the rest of the channels.

Commit:
Creating new files for guildlines.

Commit:
Corrected spelling of "accolades"

Commit:
Added description for #chat

Commit:
Added description for #off-topic

Commit:
Updated description for #off-topic

hybotics commented:
Added some more channels and descriptions to README.md. Also added new files contribute-github.md and contribut-discord.md.

Commit:
New changes

Commit:
Signed-off-by: Dale Weber <dalew@hybotics.io>

New commit

Commit:
Signed-off-by: Dale Weber <dalew@hybotics.io>

Added a section to describe how to create a GPG key and set it for signing commits.

Commit:
Signed-off-by: Dale Weber <dalew@hybotics.io>

Added some text on contributing to our Discord.
